### DISCUSSION TITLE
How to change the password of a logged in user (urgent)

### DISCUSSION THREAD

**Discussion by KetulCodiste** [Web3Auth/Web3Auth#632] [Help]
I am using Web3Auth + Auth0.
If my user want to change the password then I need to provide the change password button in my website. 
Now I want to redirect my user to the change password page of auth0. The user will be logged in when he tries to change the password.

how should i achieve this?

I am using the Universal login flow. I am not using Authentication api

**Comment by rinzler99** [comment]
https://github.com/orgs/Web3Auth/discussions/574 pls refer to this answer

**Reply by KetulCodiste** [reply]
That answer is using the authentication api. I am using the universal login flow. Plus, I am using the web3Auth plus auth0, so won't be able to use the solution that you provided.
Can you please share me the flow of how to directly redirect the user to the forgot password / change password page in "universal login"

**Reply by rinzler99** [reply]
Are you not using custom authentication? I guess you are web3authCore+auth0 right?

**Reply by KetulCodiste** [reply]
yes web3authcore + auth0.

**Reply by rinzler99** [reply]
https://auth0.com/docs/authenticate/database-connections/password-change
<img width="865" alt="image" src="https://user-images.githubusercontent.com/30899350/191956800-64976a27-6874-45da-8ac8-66649086dbea.png">

**Reply by KetulCodiste** [reply]
But this option is provided in the login page. 
I would need to redirect my user to the login page and from there user would need to click "Forgot password?" button and then he would need to enter the email to change the password.

I want my user to directly redirect to the forgot password page instead of the login page.

**Comment by rinzler99** [comment]
Hey, you can create a custom page for password reset even if you are using universal screens, Pls refer the auth0 document for password reset mentioned above.
<img width="834" alt="image" src="https://user-images.githubusercontent.com/30899350/191958049-db6d0c79-c75a-4b21-91b5-fc7dbd6b3f7a.png">

**Reply by KetulCodiste** [reply]
This section is for customizing the change password page. Means I can change the css of the page. I do not think it will be helpful to solve the issue of redirectlng my user to forget password page.

**Comment by himanshuchawla009** [comment]
I think this question is not relevant with web3auth, you can simply reset password with the mechanism that auth0 provides you. Web3auth has no direct link with auth0 apis. If you think using web3auth with auth0 is blocking you from adding a feature to reset user's password, it would be great for us if you can explain a bit more. We will be happy to help and resolve it.
